workers/linux/relay_worker.py

```diff
@@ -35,15 +34,21 @@ def __init__(self, config, main_thread_running,
 
         # Dynamic Properties based on config
         self.active = False
-        self.topic = self.config.get('topic', '').replace(" ",
-                                                          "/").lower() if self.config.get(
-            'topic', None) is not None else 'mudpi/relays/' + self.key
-        self.pin_state_off = True if self.config[
-                                         'normally_open'] is not None and \
-                                     self.config['normally_open'] else False
-        self.pin_state_on = False if self.config[
-                                         'normally_open'] is not None and \
-                                     self.config['normally_open'] else True
+        self.topic = (
+            self.config.get('topic', '').replace(" ", "/").lower()
```

Comment:
Maybe just as a suggestion you may want to use this https://github.com/mudpi/mudpi-core/blob/feature/utils.py#L17-L35. So we gather all configuration items the same way. 

Note that you can also use the replace as well.

Reply:
The diff showed here doesn't seem to be accurate. I don't know why.